KsApi/models/Comment.swift

```diff
@@ -24,34 +23,19 @@ extension Comment: Swift.Decodable {
     self.author = try values.decode(Author.self, forKey: .author)
     self.body = try values.decode(String.self, forKey: .body)
     self.createdAt = try values.decode(TimeInterval.self, forKey: .createdAt)
-    self.deletedAt = try values.decode(TimeInterval?.self, forKey: .deletedAt)
+    // Decode a time interval so that non-positive values are coalesced to `nil`. We do this because the API
+    // sends back `0` when the comment hasn't been deleted, and we'd rather handle that value as `nil`.
+    if let value = try values.decodeIfPresent(TimeInterval.self, forKey: .deletedAt), value > 0 {
```

Comment:
Is it possible to simplify/re-factor this with a nil-coalescing operator?
e.g. `self.deletedAt = try values.decodeIfPresent(TimeInterval.self, forKey: .deletedAt)` ?? nil

Reply:
we need to have `value > 0` check in place, and i don't know the way to make it work with nil-coalescing operator